src/sentry/integrations/slack/integration.py

```diff
@@ -157,3 +165,159 @@ def build_integration(self, state):
                 "data": {},
             },
         }
+
+        # if we have the integration_id then we need to set the
+        # information for the migration, the user_id and channels
+        # are using in post_install to send messages to slack
+        if state.get("integration_id"):
+            metadata["installation_type"] = "migrated_to_bot"
+
+            post_install_data = {
+                "user_id": state["user_id"],
+                "channels": state["private_channels"],
+            }
+
+            integration["integration_id"] = state.get("integration_id")
+            integration["post_install_data"] = post_install_data
```

Comment:
@MeredithAnya Is this adding column `post_install_data`?

Reply:
no its just data that gets passed to `_finish_pipeline`, it's not saved anywhere. maybe naming the object `integration` is confusing?

Reply:
Confusing but it's fine